modules/EasyAbp.EShop.Stores/src/EasyAbp.EShop.Stores.Application/EasyAbp/EShop/Stores/Stores/StoreAppService.cs

```diff
@@ -11,16 +10,45 @@ public class StoreAppService : CrudAppService<Store, StoreDto, Guid, PagedAndSor
         IStoreAppService
     {
         private readonly IStoreRepository _repository;
+        private readonly IStoreManager _storeManager;
 
-        public StoreAppService(IStoreRepository repository) : base(repository)
+        public StoreAppService(IStoreRepository repository,
+            IStoreManager storeManager) : base(repository)
         {
             _repository = repository;
+            _storeManager = storeManager;
         }
 
         public async Task<StoreDto> GetDefaultAsync()
         {
             // Todo: need to be improved
             return ObjectMapper.Map<Store, StoreDto>(await _repository.FindDefaultStoreAsync());
         }
+
+        public override async Task<StoreDto> CreateAsync(CreateUpdateStoreDto input)
+        {
+            await CheckCreatePolicyAsync();
+
+            var entity = MapToEntity(input);
+
+            TryToSetTenantId(entity);
+
+            entity = await _storeManager.CreateAsync(entity, input.OwnerIds);
+
+            return MapToGetOutputDto(entity);
+        }
+
+        public override async Task<StoreDto> UpdateAsync(Guid id, CreateUpdateStoreDto input)
+        {
+            await CheckUpdatePolicyAsync();
+
+            var entity = await GetEntityByIdAsync(id);
+
+            MapToEntity(input, entity);
+
+            entity = await _storeManager.UpdateAsync(entity, input.OwnerIds);
```

Comment:
The best practice of domain services is only creating/updating data of entities and use the repositories to persist the data in application services.